src/components/App.js

```diff
@@ -49,41 +41,77 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
           </div>
         </div>)
     } else {
-      optionBulletForm.push(<RadioButtonOptions key={title} options={options[chosenType][title]} setOptions={setOptions} 
-      selectedOption={info[title]}
-      title={title}/>);
-    }}
-
+      optionBulletForm.push(
+      <div key={title} className="option">
+        <div className="option-title">{title}</div>
+        <div className="option-menu">
```

Comment:
could we try putting option-menu classname directly into the Dropdown component?